lib/dockable.rb

```diff
@@ -4,15 +4,25 @@
 
 module Dockable
 
-  def self.load_environment_variables(bucket:, key:)
-    if presence(bucket) && presence(key)
+  def self.load_environment_variables(bucket:, key:, ssm_prefix:)
+    if presence(ssm_prefix)
+      require "aws-sdk-ssm"
+      client     = Aws::SSM::Client.new
+      ssm_prefix = "/#{ssm_prefix}" if ssm_prefix[0] != "/" # get_parameters_by_path requires leading /
+      parameters = client.get_parameters_by_path(path: ssm_prefix).parameters
+      if parameters.count != 0
+        parameters.map{|param| [param.name.gsub("#{path}/", "").upcase, param.value]}.to_h
```

Comment:
I might be missing it, but where is `path` coming from?

Why not `param.name.split("/").last.upcase` ?

Reply:
ah yeah you're right, i meant to do `ssm_prefix` instead of path. as for second part, i thought it was possible to have `/` in env vars but just tested it out and turns out its not right, was figuring it wouldn't hurt to allow for that but can change that bit